integration-tests/src/kusama_test_net.rs

```diff
@@ -191,6 +191,12 @@ pub fn basilisk_ext() -> sp_io::TestExternalities {
 			(b"MOVR".to_vec(), 1_000u128),
 			(b"NEW_BOOTSRAPPED_TOKEN".to_vec(), 1_000u128),
 		],
+		asset_ids: vec![
+			(b"KSM".to_vec(), 1_000_000u128, 1),
+			(b"aUSD".to_vec(), 1_000u128, 2),
+			(b"MOVR".to_vec(), 1_000u128, 3),
+			(b"NEW_BOOTSRAPPED_TOKEN".to_vec(), 1_000u128, 4),
+		],
```

Comment:
Same comment as in another PR for hydradx from Martin: Tokens should be in `asset_names` OR `asset_ids`, otherwise it will be registered twice. Sorry for making that non-obvious in my changes to the pallet.

Reply:
Ok. I'm going to update the pallet so we can include that change in this PR.